latest/index.html

```diff
@@ -558,7 +559,7 @@ <h3>Preview Metadata</h3>
       <p>Reconciliation services MAY offer a preview service by providing the <dfn>preview metadata</dfn> as an object stored in the <a>service manifest</a> under the key <code>preview</code>. It consists of the following fields, all mandatory:
         <dl>
           <dt><code>url</code></dt>
-          <dd>A string containing <code>{{id}<!-- -->}</code> such that replacing <code>{{id}<!-- -->}</code> by an <a>entity</a> identifier yields the <dfn>preview URL</dfn> for that entity. This preview URL MUST resolve to an HTML page summarizing the entity.
+          <dd>A string containing <code>{{id}<!-- -->}</code> such that replacing <code>{{id}<!-- -->}</code> by an <a>entity</a> identifier encoded as a URI component yields the <dfn>preview URL</dfn> for that entity. This preview URL MUST resolve to an HTML page summarizing the entity.
```

Comment:
Should we say "percent-encoded" or just "encoded" here?

Reply:
I think "encoded as a URI component" is quite precise, no?

Reply:
My idea was to have the details in the first mention in the entity section (which contains "percent-encode"), basically defining what "encode as a URI component" means, and then using that short form.

Reply:
@fsteeg OK sure no problem. RFC3986 continually used the long form.